apps/internal/oauth/ops/accesstokens/accesstokens.go

```diff
@@ -208,7 +208,7 @@ func (c Client) FromAuthCode(ctx context.Context, req AuthCodeRequest) (TokenRes
 			return TokenResponse{}, err
 		}
 	case ATPublic:
-		// Nothing needs to be done, exept to not error.
+		qv = url.Values{}
```

Comment:
Is this different that what is declared on line 196 ?

Reply:
Yeah, that tripped me up when I saw it.  

Yes, 196 has a value of nil.
211 has a value of url.Values{} with nothing in it.

So, if the using function got nil and did this:
qv.Get("key")
it would crash.

if it got an empty map, it would not.